spec/models/post_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+RSpec.describe Post, type: :model do
+  describe 'validations' do
+    it 'requires a title' do
+      post = Post.new(title: nil)
+      post.valid?
+      expect(post.errors[:title]).to include("can't be blank")
+    end
+    it 'validates the maximum length of title' do
+      post = Post.new(title: 'a' * 251)
+      post.valid?
+      expect(post.errors[:title]).to include('is too long (maximum is 250 characters)')
+    end
```

Comment:
- [ ] Please write a test to cover the `five_most_recent_comments` method in the `Post` model